src/ingestor.ts

```
@@ -79,6 +80,16 @@ export default async function ingestor(req) {
     if (!Object.keys(hashTypes).includes(hash)) return Promise.reject('wrong types');
     type = hashTypes[hash];
 
+    try {
+      if (
+        (message.space.startsWith('s:') || !message.space.includes(':')) &&
```

Not sure how we missed before but it should throw an error if `message.space` is undefined no?

---

I believe the schema validation is already ensuring `message.space` is always set

---

No 🙈 it is optional. but something else is preventing it not to throw an error 🤔

---

I will run few tests and see. but else it looks good, we can merge

---

I tried to edit profile with this change (alias request) and it fail 
```ts
undefined
TypeError: Cannot read properties of undefined (reading 'startsWith')
```

---

how it was working before 😅 with https://github.com/snapshot-labs/snapshot-sequencer/pull/417/files

---

Maybe we could skip the check if `!message.space`